streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1752,7 +1752,18 @@ public void shouldCloseStateManagerEvenDuringFailureOnUncleanTaskClose() {
     }
 
     @Test
-    public void shouldReturnOffsetsForRepartitionTopicsForPurging() {
+    public void shouldReturnCommittedOffsetsForRepartitionTopicsForPurging() {
+        final Map<TopicPartition, Long> purgeableOffsets = doReturnPurgeableOffsets(true);
+        assertThat(purgeableOffsets, equalTo(singletonMap(new TopicPartition("repartition", 1), 10L)));
+    }
+
+    @Test
+    public void shouldReturnEmptyMapWhenNoCommitForRepartitionTopicsForPurging() {
+        final Map<TopicPartition, Long> purgeableOffsets = doReturnPurgeableOffsets(false);
+        assertThat(purgeableOffsets, equalTo(Collections.emptyMap()));
+    }
```

Comment:
Needed to do this to re-use code as Junit4 doesn't have `@Parameterized` at the test level.